docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst

```diff
@@ -0,0 +1,55 @@
+.. _pro-check-list-cves:
+
+How to know if a system is affected by this list of CVEs?
+*********************************************************
+
+.. include:: ../includes/pro-fix-intro.txt
+
+If you've found a `Common Vulnerabilities and Exposure <cve_>`_ (CVEs) and `Ubuntu Security Notice <usn_>`_ (USNs) and want to check if your Ubuntu system is affected by it, you can check using the ``pro fix`` command as we'll show you in this guide.
+
+.. note::
+    The ``pro fix`` API is provided as a part of the Ubuntu Pro Client (``pro``), which is a security tool for Ubuntu systems. The Ubuntu Pro Client comes pre-installed on every Ubuntu system. You can run ``pro help`` in your terminal window to see a list of the ``pro`` services and commands available.
```

Comment:
The `pro fix` is not an API. I think we should adapt this part

Reply:
Or we need to better detail the API endpoint we are talking about

Reply:
Oh I see, I am using `pro fix` command almost everywhere else so to keep it consistent, I can change the API here to command. It would be better

Reply:
Should be updated @lucasmoura